### PR TITLE
Expose FormatFile and FormatSnippet.

### PR DESCRIPTION
**Pull request by absoludity**
Not sure if I should be updating the jsonnet.Version() result here, based on the comment, I assume so.

### CODE REVIEW

File: jsonnet_test.go [hunk 1]
```diff
@@ -105,3 +107,67 @@ func Test_Misc(t *testing.T) {
...
+	if err != nil {
+		t.Fatal(err)
+	}
+	filename := f.Name()
+	data := `{
+    "quoted": "keys",
+    "notevaluated": 20 + 22,
+    "trailing": "comma",}
+`
+	if err := ioutil.WriteFile(filename, []byte(data), 0644); err != nil {
```
Comment (mmikulicic) [inline jsonnet_test.go]: ioutil.TempFile already opens the file. Let's also make sure we cleanup the temp file even if there is a write error.

i.e.:

```
defer func() {
  f.Close()
  os.Remove(filename)
}()

data := ....

if _, err := f.Write(data); err != nil {
 t.Fatal("Writing to %q: %v", f.Name, err)
}
```

File: jsonnet_test.go [hunk 2]
```diff
@@ -105,3 +107,67 @@ func Test_Misc(t *testing.T) {
...
+
+func Test_FormatIndent(t *testing.T) {
+	data := `{
+  "quoted": "keys",
+ "notevaluated": 20 + 22,
+   "trailing": "comma",}
+`
+
+	vm := jsonnet.Make()
+	vm.FormatIndent(4)
```
Comment (mmikulicic) [inline jsonnet_test.go]: isn't 4 already the default indent? shouldn't we try another number?

File: jsonnet_test.go (hunk 1 above)
Comment (absoludity) [inline jsonnet_test.go]: Right, moved - thanks.

File: jsonnet_test.go (hunk 2 above)
Comment (absoludity) [inline jsonnet_test.go]: Nope, by default it doesn't adjust the indent at all (ie. if you comment out the call to vm.FormatIndent(), the indentation matches the initial string (though other things are formatted, such as keys not needing quotes). But I'll update it to indent 1 just to make it more obvious that it's not a default.

**Pull request merged**